libraries/AP_LTM_Telem/AP_LTM_Telem.h

```diff
@@ -0,0 +1,57 @@
+/*
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+#define LTM_PAYLOAD_STATUS_CAPACITY               5    // size of the message buffer queue (max number of messages waiting to be sent)
```

Comment:
Nothing but AP_LTM_Telem.cpp needs to know about these defines.  The could be moved into their to avoid polluting other namespaces.  Particularly important for things as generic as `YAW_LIMIT`.

Additionally, I've got a preference for actually using static const types constants in place of defines - static type checking is good!  That being said, it's not a blocker.

Reply:
updated